### PR TITLE
lintian: override invalid error where lintian doesn't evaluate second debhelper dep

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?

This overrides an invalid lintian error. We'd like to fix this for version 28.

## Test Steps

Build and run lintian. Make sure there are no errors on jammy++

## Checklist

 - [n/a] I have updated or added any unit tests accordingly
 - [n/a] I have updated or added any integration tests accordingly
 - [n/a] I have updated or added any documentation accordingly

## Does this PR require extra reviews?

 - [x] Yes
 - [ ] No

### CODE REVIEW

File: debian/source/lintian-overrides [hunk 1]
```diff
@@ -2,5 +2,8 @@
 ubuntu-advantage-tools: missing-build-dependency-for-dh_-command dh_systemd_start => dh-systemd
 ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd => dh-systemd
 
+# Lintian doesn't see the second debhelper dependency with >13.3 on jammy++
+ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd *debhelper* (>= 9.20160709~)*
+
 # Lintian doesn't like mentioning riscv64 for older go package
```
Comment (renanrodrigo) [inline debian/source/lintian-overrides:8]: Question: do we need this override? we do not go anymore do we?

Comment (orndorffgrant) [reply debian/source/lintian-overrides:8]: I've created a follow up issue to look at all warnings and also remove that line here #2608

Comment (orndorffgrant) [reply debian/source/lintian-overrides:8]: This PR is just a minimal diff to override the error for release 28

Comment (renanrodrigo) [reply debian/source/lintian-overrides:8]: aha nice good good

File: debian/source/lintian-overrides [hunk 2]
```diff
@@ -2,5 +2,8 @@
 ubuntu-advantage-tools: missing-build-dependency-for-dh_-command dh_systemd_start => dh-systemd
 ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd => dh-systemd
 
+# Lintian doesn't see the second debhelper dependency with >13.3 on jammy++
+ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd *debhelper* (>= 9.20160709~)*
```
Comment (lucaskanashiro) [inline debian/source/lintian-overrides:6]: This is correct and overrides the lintian error. There is an alternative that would avoid the almost duplicate with line 3:

```
ubuntu-advantage-tools: missing-build-dependency-for-dh-addon  *systemd* 
```

With that, line three could be removed and this new override would cover both cases. I understand you want to minimize the changes here to release version 28, but I thought I should at least raise this here :)

Feel free to land this as you proposed and fix it later as part of #2608 .

**Review by lucaskanashiro — APPROVED** [review]

File: debian/source/lintian-overrides (hunk 2 above)
Comment (orndorffgrant) [reply debian/source/lintian-overrides:6]: This is closely related enough I think we can include it now! I'll test this locally and update the PR with it

Comment (orndorffgrant) [reply debian/source/lintian-overrides]: @lucaskanashiro updated!

Comment (lucaskanashiro) [reply debian/source/lintian-overrides]: Perfect, thanks @orndorffgrant !

**Pull request merged**